google_maps/public/html/google_maps_edit.html

```diff
@@ -17,50 +17,62 @@ <h2 class="sr">{% trans "XBlock settings" %}</h2>
         <span class="tip setting-help" id="display-name-help">{{ fields.display_name.help }}</span>
       </li>
 
-      <li class="field comp-setting-entry is-set edit-api-key-container">
-        <div class="wrapper-comp-setting">
-          <label class="label setting-label" for="api-key-input">{{ fields.api_key.display_name }}</label>
-          <input class="input setting-input edit-api-key" id="api-key-input" value="{{ self.api_key }}" type="text">
-        </div>
-      </li>
+      {% if self.api_key %}
+      <div class="google-maps-container">
+        <div class="google-maps-inputs">
+          <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+              <label class="label setting-label" for="place-name-input">{% trans "Search for a place" %}</label>
+              <input class="input setting-input edit-place-name" id="place-name-input" value="{{ self.place_name }}" type="text"
+              placeholder="{% trans 'Enter a location' %}" data-default-name-value="{{ fields.place.default.name }}"
+              data-default-id-value="{{ fields.place.default.id }}">
+              <button class="action setting-clear clear-iframe-place-name" type="button" name="setting-clear">
+                <i class="icon fa fa-undo" aria-hidden="true"></i>
+                <span class="sr">{% trans "Reset iframe place name to its default value" %}</span>
+              </button>
+              <p class="error-message hidden">
+                  <i class="fa fa-times-circle" aria-hidden="true"></i>
```

Comment:
FYI this is the version we need http://fontawesome.io/icon/exclamation-triangle/